drizzlepac/haputils/product.py

```diff
@@ -868,6 +868,7 @@ def wcs_drizzle_product(self, meta_wcs):
         drizzle_pars["preserve"] = False
         drizzle_pars['rules_file'] = self.rules_file
         drizzle_pars['resetbits'] = "0"
+        drizzle_pars['context'] = False
```

Comment:
Please add a comment here noting the reason for setting this parameter as you have done for the code below.